extensions/wikia/GlobalNavigation/scripts/GlobalNavigationAccountNavigation.js

```diff
@@ -87,5 +84,38 @@ require([
 					scrollFix.restoreScrollY($globalNavigation);
 				});
 		}
+	}
+
+	$(function () {
+		var $entryPoint, $authEntryPoints;
+
+		$entryPoint = $('#AccountNavigation');
+
+		if (!win.wgUserName && $entryPoint.hasClass('newAuth')) {
+			$authEntryPoints = $('.auth-link.register, .auth-link.sign-in, a.sign-in');
```

Comment:
Are we sure we want to replace functionality on all ```.sign-in``` links from the beginning?
I suggest binding only to global nav at start.